contracts/protocol/bases/SellerBase.sol

```diff
@@ -49,6 +49,40 @@ contract SellerBase is ProtocolBase, IBosonAccountEvents {
             ADMIN_OR_AUTH_TOKEN
         );
 
+        // Cache protocol lookups for reference
+        ProtocolLib.ProtocolLookups storage lookups = protocolLookups();
+
+        // Get message sender
+        address sender = msgSender();
+
+        // Check that caller is the supplied operator and clerk
+        require(_seller.operator == sender && _seller.clerk == sender, NOT_OPERATOR_AND_CLERK);
```

Comment:
Why the check for treasury isn't done:
- frontrunner cannot create seller or DR with fields {admin: victim, operator: victim, clerk: victim, treasury: attacker}. It will fail, since new check will detect that msg.sender (==atacker) does not match the victim's address
- treasury cannot perform any action in the protocol